CMakeLists.txt

```
@@ -0,0 +1,36 @@
+################################################################################
+#    Copyright (C) 2021 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    #
+#                                                                              #
+#              This software is distributed under the terms of the             #
+#              GNU Lesser General Public Licence (LGPL) version 3,             #
+#                  copied verbatim in the file "LICENSE"                       #
+################################################################################
+
+cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
+cmake_policy(VERSION 3.14...3.20)
+
+project(FairCMakeModules VERSION 1.0 LANGUAGES NONE)
```

```suggestion
project(FairCMakeModules VERSION 1.0.0 LANGUAGES NONE)
```

Let's make it 3 component version following and add a corresponding git tag with `v` prefix once this PR is merged.

---

done.

Also added printing of the Version during Configure time.